test/Infer/no-const-inputs.opt

```diff
@@ -1,8 +1,6 @@
 ; REQUIRES: solver
 ; RUN: %souper-check -infer-rhs %solver -souper-infer-inst -souper-synthesis-comps=add,const %s > %t1
 ; RUN: %FileCheck %s < %t1
-; RUN: %souper-check -infer-rhs %solver -souper-exhaustive-synthesis %s > %t2
-; RUN: %FileCheck %s < %t2
```

Comment:
when you want to keep a test case, but it is expected to fail, you should not remove important parts of it, since we won't remember what do add back later

just mark it as XFAIL